### ISSUE TITLE
In-hospital emergency rooms should not be areas

### ISSUE THREAD

**Issue by StormeHawke** [WazeUSA/WME-Place-Harmonizer#145]
This might be tricky to fix.

Per this wiki (which while it is a user page I've been told it's basically official guidance)
https://wiki.waze.com/wiki/User:Fredo-p/Hospitals

> Mapping Emergency Rooms (ER)
> Properly mapping a hospital's ER is very important for both routing and search results. The place point of the ER should be located at the public entrance of the ER. If the facility has a website map, it will usually indicate the location of the ER entrance.
> In some cases, hospitals may operate emergency centers at locations other than the hospital itself, especially in rural areas across the US. These locations only serve one purpose; emergency care -- the entire building is an ER. In this case, map the building as an Area, and place the stop point at the patient entrance.

So basically an ER should be a point any time it is located within a Hospital area place, but an area point otherwise.  But should still be categorize as Hospital / Medical care.

Here is an example of an ER place that is correctly marked as a point within a larger hospital area:
https://www.waze.com/editor/?env=usa&lon=-85.66660&lat=40.56388&zoom=7&venues=179765654.1797853144.855011

**Comment by t0cableguy** [comment]
IE Detect ER or Emergency Room and set up wmeph to allow a point.

**Comment by StormeHawke** [comment]
Yes, except that it can only be a point if it's part of a hospital area.  It has to be an area if it's one of those off-site ERs

**Comment by StormeHawke** [comment]
I suppose a user message to check which type it should be would be appropriate as long as it goes away once the place is locked to regional standards
